.ci/generate-test-mech.py

```diff
@@ -0,0 +1,10 @@
+import cantera as ct
+import pyrometheus as pyro
+import glob
+
+test_mechs = glob.glob("test/mechs/*.cti")
+for mech in test_mechs:
+    mechname = mech[11:-4]
+    with open(f"test/mechs/{mechname}_mech.py", "w") as outf:
```

Comment:
The sphinx bits in #12 rely on a mechanism appearing at `pyrometheus.thermochem_example`. (But I realize there are multiple mechanisms here. Maybe emit the first one into `pyrometheus.thermochem_example`, too?

Reply:
Oh sorry, I did not see that.  I'll get that fixed up.

Longer term, it might be nice if we had some doc support for the general interface, and then a little blurb about each mechanism - maybe from embedded documentation, in which case we'd want to have all of them represented in actual py code?

Reply:
I refactored it just a bit with a function that generates a specified mech py file from a specified cti file. See the utils in `.ci` and `.github/workflows` yml file `doc` action for the generation of the required example.

Reply:
ab15da3, 68a9507